worlds/cvcotm/options.py

```diff
@@ -0,0 +1,160 @@
+    option_hard = 2
+    default = 1
+
+
+class HalveDSSCardsPlaced(Toggle):
+    """Places only half of the DSS Cards in the item pool.
+    A valid combo that lets you freeze or petrify enemies as platforms will always be placed."""
+    display_name = "Halve DSS Cards Placed"
+
+
+class Countdown(Choice):
+    """Displays, below and near the right side of the MP bar, the number of unobtained progression/useful-marked
+    items or the total check locations remaining in the area you are currently in."""
+    display_name = "Countdown"
+    option_none = 0
+    option_majors = 1
+    option_all_locations = 2
+    default = 0
+
+
+class SubWeaponShuffle(Toggle):
+    """Randomizes which sub-weapon candles have which sub-weapons.
+    The total count of each sub-weapon will be consistent with that of the vanilla game."""
+    display_name = "Sub-weapon Shuffle"
+
+
+class DisableBattleArenaMPDrain(Toggle):
+    """Makes the Battle Arena not drain Nathan's MP, so that DSS combos can be freely used."""
+    display_name = "Disable Battle Arena MP Drain"
+
+
+class RequireAllBosses(Toggle):
+    """Forces a Last Key behind every boss and requires all 8 of them to enter the Ceremonial Room.
+    The Required and Available Last Keys settings will both be forced to 8."""
+    display_name = "Require All Bosses"
+
+
+class EarlyDouble(DefaultOnToggle):
+    """Ensures the Double will be placed somewhere within the Catacombs in your own game, accessible with nothing."""
+    display_name = "Early Double"
+
+
+class DeathLink(Choice):
+    """When you die, everyone dies. Of course the reverse is true too.
+    Received DeathLinks will not kill you in the Battle Arena unless Arena On is chosen."""
+    display_name = "DeathLink"
+    option_off = 0
+    alias_no = 0
+    alias_true = 1
+    alias_yes = 1
+    option_on = 1
+    option_arena_on = 2
+    default = 0
+
+
+class CompletionGoal(Choice):
+    """The goal for game completion. Whether it be defeating Dracula, winning in the Battle Arena, or both."""
+    display_name = "Completion Goal"
+    option_dracula = 0
+    option_battle_arena = 1
+    option_battle_arena_and_dracula = 2
+    default = 0
+
+
+@dataclass
+class CVCotMOptions(PerGameCommonOptions):
+    ignore_cleansing: IgnoreCleansing
+    auto_run: AutoRun
+    dss_patch: DSSPatch
+    always_allow_speed_dash: AlwaysAllowSpeedDash
+    break_iron_maidens: BreakIronMaidens
+    required_last_keys: RequiredLastKeys
+    available_last_keys: AvailableLastKeys
+    buff_ranged_familiars: BuffRangedFamiliars
+    buff_sub_weapons: BuffSubWeapons
+    buff_shooter_strength: BuffShooterStrength
+    item_drop_randomization: ItemDropRandomization
+    halve_dss_cards_placed: HalveDSSCardsPlaced
+    countdown: Countdown
+    sub_weapon_shuffle: SubWeaponShuffle
+    disable_battle_arena_mp_drain: DisableBattleArenaMPDrain
+    require_all_bosses: RequireAllBosses
+    early_double: EarlyDouble
+    start_inventory_from_pool: StartInventoryPool
```

Comment:
I generally recommend putting start_inventory_from_pool at the top, as that would put it next to the standard archipelago options in the yaml. Or putting it at the bottom.